__tests__/api.test.js

```diff
@@ -0,0 +1,26 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api", () => {
+    it("will return 200 status code", () => {
+        return request(app)
+        .get("/api")
+        .expect(200);
+    })
+    it("will return 200 and the JSON of all available endpoints", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body).toHaveProperty("GET /api")
```

Comment:
This test should be dynamic enough not to have to update it with each new endpoint added to the json file. Therefore, you can just require in your endpoints file and expect what's returned to match it.